.github/workflows/molecule.yml

```diff
@@ -0,0 +1,42 @@
+---
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          path: "${{ github.repository }}"
+      - name: molecule
+        uses: robertdebock/molecule-action@2.6.3
+        with:
+          command: lint
+  test:
+    needs:
+      - lint
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
```

Comment:
When setting up the test skeleton, please have a look at the Readme and the meta/main.yml - if the role looks like it advertises support for both red hat and ubuntu, we make sure to test both platforms in the test matrix.

Reply:
(by adding GG's Centos 7 + 8 images to the list of tested images)

Reply:
I added some functional tests to verify.yml, so as long as nothing breaks from those, this can all get merged in once the centos tests pass.